LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```

Comment:
Let's remove this one, we already have `LICENSE` file.

Reply:
nice catch 👍